tensorflow/tensorboard
# data: require `ctx` and fix all call sites

Summary:
This patch changes the `MultiplexerDataProvider` to check that the `ctx`
argument is always passed, and fixes all call sites to pass it.

Test Plan:
Manually verified behavior of the following plugins:

  - scalars, incl. download links
  - custom scalars, incl. download links
  - images
  - audio
  - graphs (`--generic_data=true` required)
  - distributions
  - histograms
  - text
  - PR curves
  - hparams (automated tests do not suffice for any functionality)

Other plugins (debugger, profile, beholder, mesh, what-if tool) don’t
use the data provider interface and so are unaffected by this change.
The `text_v2` plugin does, but with `?experimentalPlugin=text_v2` I only
see a blank screen, even at head.

wchargin-branch: data-ctx-required
